pylearn2/datasets/mnist_augmented.py

```diff
@@ -0,0 +1,80 @@
+"""
+Augmented MNIST wrapper class
+"""
+
+import os
+import numpy as np
+
+from pylearn2.datasets.dense_design_matrix import DenseDesignMatrix
+from pylearn2.scripts.dbm.augment_input import augment_input
+from pylearn2.utils import serial
+
+
```

Comment:
We only use all caps when it's actually an acronym. I would suggest something like AugmentedMNIST.